fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3433,6 +3433,12 @@ private DataProperty analyzeMVDataProperties(Database db,
                 setLakeStorageInfo(materializedView, storageVolumeId, properties);
             }
 
+            // datacache.partition_duration
```

Comment:
> what's the behavior if the property is not set? will it have a default value or something else, can this behavior be covered by some UTs as well?

If this parameter is not set, it will have no default value and tableProperty.dataCachePartitionDuration will remain null. The above content is covered in OlapTableTest.testNullDataCachePartitionDuration.

Reply:
> OlapTableTest.testNullDataCachePartitionDuration

does it also cover mv case? lke if mv is not provided with datacache partition duration, it will be null and the check shouldCache to the partition are always to `false` result?

Reply:
> > what's the behavior if the property is not set? will it have a default value or something else, can this behavior be covered by some UTs as well?
> 
> If this parameter is not set, it will have no default value and tableProperty.dataCachePartitionDuration will remain null. The above content is covered in OlapTableTest.testNullDataCachePartitionDuration.

MaterializedView is a subclass of OlapTable. Mv's OlapTable.tableProperty.dataCachePartitionDuration is set in analyzeMVDataProperties. All entries for using datacache are the same: olapTable.isEnableFillDataCache (in OlapScanNode.addScanRangeLocations)